packages/opencensus-exporter-stackdriver/src/types.ts

```diff
@@ -151,6 +151,11 @@ export interface StackdriverExporterOptions extends ExporterConfig {
    * of a stackdriver metric. Optional
    */
   prefix?: string;
+  /**
+   * Create a JWT instance using the given credentials input containing
+   * client_email and private_key properties. Optional
```

Comment:
Can you update this field comment with: "If this field is set, its contents will be used for authentication instead of your application default credentials."?

Reply:
OK! Thanks for your suggestion.